frontend/src/main/web/src/components/Flex/readme.md

```diff
@@ -0,0 +1,2 @@
+For a more comprehensive guide on how Flexbox works, check out
+[A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).
```

Comment:
So is this component just a Flexbox maker? I can't see that stated anywhere. Is this a component we should use directly or is it supposed to be inherited?